### DISCUSSION TITLE
Updating an existing transaction, "foreign_currency_id" gives 422

### DISCUSSION THREAD

**Discussion by adyanth** [firefly-iii/firefly-iii#6827] [Firefly III] [answered]
Doing the same as https://github.com/orgs/firefly-iii/discussions/6820, while updating a transaction after getting it from firefly, firefly complains about the `foreign_currency_id` field being set to 0. Refer the JSON in the linked discussion, this is being returned by firefly itself.

Deleting the field before the PUT call works well, but this looks to be an incorrect API behavior?

**Comment by JC5** [comment, marked as answer]
I see what you mean. Same with the decimal_places. I believe both should be null.

**Reply by JC5** [reply]
https://github.com/firefly-iii/firefly-iii/issues/6828
